### ISSUE TITLE
ETDMS namespace and schema

### ISSUE THREAD

**Issue by pbinkley** [ualbertalib/fcrepo4-oaiprovider#9]
The namespace and schema we report for ETDMS records are out of line with other LAC-harvested repos. Let's go with:

```
<metadataFormat>
    <metadataPrefix>oai_etdms</metadataPrefix>
    <schema>http://www.ndltd.org/standards/metadata/etdms/1.0/etdms.xsd</schema>
    <metadataNamespace>http://www.ndltd.org/standards/metadata/etdms/1.0/</metadataNamespace>
</metadataFormat>
```

Drawn from https://ir.library.dc-uoit.ca/dspace-oai/request?verb=ListMetadataFormats, provided as a model by LAC.

**Comment by jefferya** [comment]
@pbinkley 

Two questions.

(1) I remembered a previous issue #5 where we changed the schema due to an error from the "http://validator.oaipmh.com" - "Schema http://www.ndltd.org/standards/metadata/etdms/1.0/xml.xsd not found". The above proposed replacement, yields a page not found error. Is this ok in the LAC context?

(2) Is it right to assume the the "schemaLocation" attribute should also be changed? For example, 
```
xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd http://www.openarchives.org/OAI/2.0/oai-identifier http://www.openarchives.org/OAI/2.0/oai-identifier.xsd http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd http://www.ndltd.org/standards/metadata/etdms/1.0/ http://www.ndltd.org/standards/metadata/etdms/1-0/etdms.xsd"
```

**Comment by pbinkley** [comment]
Per our discussion just now:

1. We can live with validator errors to make it work for LAC
2. Yes, sorry, I should have specified: let's be consistent in all uses of the namespace

**Comment by pbinkley** [comment]
Closed by #10
